charts/argocd-apps/Chart.yaml

```diff
@@ -17,5 +17,12 @@ annotations:
     fingerprint: 2B8F22F57260EFA67BE1C5824B11F800CD9D2252
     url: https://argoproj.github.io/argo-helm/pgp_keys.asc
   artifacthub.io/changes: |
+    - kind: added
+      description: make the chart use maps instead of lists
+      links:
+        - name: GitHub Issue
+          url: https://github.com/argoproj/argo-helm/issues/2523
+        - name: GitHub PR
+          url: https://github.com/argoproj/argo-helm/pull/2538
```

Comment:
Can you remove the links, we do not use them :-)

Also please drop the lines 27 and 28 (only the annotations from the current PRs needs to stay here)

Reply:
Hi, done as requested

Reply:
Lines below your change (L22/23) still needs to be removed

Reply:
Hi @mkilchhofer done as requested